Makefile

```diff
@@ -1,26 +1,28 @@
 NAME=hg-agent
-VERSION=1.18
+VERSION=2.0
 ARCH=amd64
 
 docker:
-	docker build -t hostedgraphite/hg-agent-build .
+	docker build -t hostedgraphite/hg-agent-build-os7-py3 .
```

Comment:
May I ask you to share with me some details on this `os7-py3` suffix? Does it have anything to do with Centos 7 OS? How did we get this image?

Reply:
This image was created on my local machine and pushed to the DockerHub.
![image](https://user-images.githubusercontent.com/40685761/155692428-e232d475-99d3-4290-a850-58211f95d7d0.png)

Reply:
It uses Dockerfile from hg-agent folder.

Reply:
Brilliant! Thanks for the explanation